### ISSUE TITLE
iRedMail 0.9.4 doesn`t support Ubuntu 16.04 (current phusion/baseimage)

### ISSUE THREAD

**Issue by approximatenumber** [velaluqa/docker-iredmail#2]
```
$ docker build -t iredmail:0.9.4 --build-arg DOMAIN={first domain} build/
...
E: Unable to locate package php5-imap
E: Unable to locate package php5-json                                                                                                                                     
E: Package 'php5-gd' has no installation candidate                                                                                                                        
E: Package 'php5-mcrypt' has no installation candidate                                                                                                                    
E: Unable to locate package php5-curl                                                                                                                                     
E: Package 'php-apc' has no installation candidate                                                                                                                        
E: Unable to locate package php5-intl                                                                                                                                     
E: Unable to locate package php5-ldap                                                                                                                                     
E: Package 'php5-mysql' has no installation candidate                                                                                                                     
E: Package 'php5-fpm' has no installation candidate                                                                                                                       
E: Unable to locate package php5-intl                                                                                                                                     
<< ERROR >> Installation failed, please check the terminal output.
<< ERROR >> If you're not sure what the problem is, try to get help in iRedMail
<< ERROR >> forum: http://www.iredmail.org/forum/
nginx: unrecognized service
clamav-freshclam: unrecognized service                                                                                                                                    
clamav-daemon: unrecognized service                                                                                                                                       
amavis: unrecognized service                                                                                                                                              
php5-fpm: unrecognized service                                                                                                                                            
mysql: unrecognized service 
...
```
iRedMail 0.9.4 [doesn\`t support](http://www.iredmail.org/forum/post48452.html#p48452) updated Ubuntu 16.04.
So, it needs to update `ENV IREDMAIL_VERSION 0.9.5` in `Dockerfile`. I will create pull request

**Comment by approximatenumber** [comment]
\+ 
iRedmail verison in `README.md` needs to be updated too ;-)

**Comment by approximatenumber** [comment]
Closed because iRedMail 0.9.5 installing fails with Ubuntu 16.04 (current phusion/baseimage).

**Comment by approximatenumber** [comment]
Reopened. 
So, iRedMail 0.9.5/0.9.4 doesn`t support Ubuntu 16.04 (current phusion/baseimage). 
I think the easest way to get working baseimage with iRedMail is to use:
- `phusion/baseimage: 0.9.18`
- `iredmail: 0.9.4`

**Comment by kajoik** [comment]
Please try out my feature branch updating the whole thing to 0.9.5-1... I haven't had the time to test it throughly.

**Comment by approximatenumber** [comment]
@franzk I checked and it failed again. It needs the time to correct the Dockerfile and to test the system. If I have it I will do it.

**Comment by kajoik** [comment]
@approximatenumber I have set a fixed tag of phusion/baseimage with 015aa2b (tag 0.9.4-3), a container with iRedMail 0.9.4 should now build with that version again.

Meanwhile, I have updated master to 0.9.5-1 and there's a feature branch for 0.9.6, too. If you get build errors with any of them, please open another issue.
